run.py

```diff
@@ -37,6 +37,8 @@ def get_args():
 options = webdriver.ChromeOptions()
 options.gpu = False
 options.binary_location = "/usr/bin/google-chrome-beta"
+options.add_argument("--enable-logging")
+options.add_argument("--v=1")
```

Comment:
What is this for?

Reply:
for debug

https://support.google.com/chrome/a/answer/6271282?hl=en

Reply:
Let's split this out into a separate PR.